cmd/lncli/commands.go

```diff
@@ -373,6 +373,22 @@ func sendCoins(ctx *cli.Context) error {
 			"sweep all coins out of the wallet")
 	}
 
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+	minConfs := int32(ctx.Uint64("min_confs"))
+
+	// In case that the user has specified the sweepall flag, we'll
+	// calculate the amount to send based on the current wallet balance.
+	if ctx.Bool("sweepall") {
+		balanceResponse, _ := client.WalletBalance(
+			ctxc, &lnrpc.WalletBalanceRequest{
+				Account:  ctx.String("account"),
+				MinConfs: minConfs,
+			},
+		)
+		amt = balanceResponse.GetConfirmedBalance()
```

Comment:
I think we should not overwrite the amount we send to the RPC, as that could have unintended side effects (especially if there are multiple accounts). So instead, this should just set a new variable, for example `displayAmt` that is only used for printing the amount below.

Reply:
Done 🏁